Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -484,6 +485,22 @@ static ExceptionOr<void> forEachRendererInPaintOrder(NOESCAPE const std::functio
     return { };
 };
 
+static bool rendererIsFragmented(const RenderLayerModelObject& renderer)
```

Comment:
how does this work for column spanners? Is `renderer` here the placeholder or the real one?

Reply:
That one I am unsure of. Would a spanner be considered fragmented in this case?

Reply:
No, it would not. It spans across all columns.

Reply:
(or you mean like would rendererIsFragmented find it fragmented?)

Reply:
I did mean the former. The same spec has this wording: "A box can be broken into multiple fragments due to [column-spanner-in-block](https://www.w3.org/TR/css-multicol-1/#spanning-columns) splitting (see [CSS Multi-column Layout](https://www.w3.org/TR/css-multicol-1/#spanning-columns))."

The linked Multicol spec isn't super clear (to me), but I think it's referring to things that get fragmented around the spanner, not the spanner itself.

- It does seem that the current code would incorrectly report spanners as fragmented though.

Reply:
Added a new test for the column spanner case!